response_operations_ui/templates/errors/500-error.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %}Server error (Error 500) - ONS Business Surveys{% endblock %}
+
+{% block main %}
+        <h1 class="saturn">Error 500 - Server error</h1>
+
+        <div class="panel panel--simple panel--error">
+
+            <p>Something has gone wrong with the website.</p>
```

Comment:
Is this the correct wording? Doesn't read that well.

Also why website and not application?

Reply:
We haven't had the final designs from RAD yet or whether the stylesheets will be pulled from elsewhere